src/test/java/org/codeseoul/event_member_management/member/MemberRestIntegrationTests.java

```diff
@@ -92,19 +94,32 @@ public void savesAMember() throws Exception {
     }
 
     @Test
-    public void updatesAMember() throws Exception {
-        aMember.setUsername("aNewUsername");
-        mockMvc.perform(put("/members/1")
+    public void updatesAnExistingMember() throws Exception {
+        aMember.setEmail("newExample@email.com");
+        mockMvc.perform(put(String.format("/members/%d", aMember.getId()))
             .contentType(MediaType.APPLICATION_JSON)
             .accept(MediaType.APPLICATION_JSON)
             .content(mapper.writeValueAsString(aMember)))
             .andExpect(status().isOk())
-            .andExpect(jsonPath("$.username", is(aMember.getUsername())));
+            .andExpect(jsonPath("$.email", is(aMember.getEmail())));
+    }
```

Comment:
Any specific reason to switch to email instead of username? Just curious.

Reply:
The username is not being set in the update method. I wasn't sure if that was intentional or not